goal_src/jak2/pc/progress/progress-static-pc.gc

```diff
@@ -310,40 +340,250 @@ This gives us more freedom to write code how we want.
         :get-value-fn (lambda () (-> *pc-settings* speedrunner-mode?))
         :on-confirm (lambda ((val symbol))
           (set! (-> *pc-settings* speedrunner-mode?) val)
-          (commit-to-file *pc-settings*))))))
+          (commit-to-file *pc-settings*)))
+        (new 'static 'menu-generic-boolean-option
+          :name (text-id progress-fast-airlock)
+          :truthy-text (text-id progress-on)
+          :falsey-text (text-id progress-off)
+          :get-value-fn (lambda () (-> *pc-settings* fast-airlock?))
+          :on-confirm (lambda ((val symbol))
+            (set! (-> *pc-settings* fast-airlock?) val)
+            (commit-to-file *pc-settings*)))
+        (new 'static 'menu-generic-boolean-option
+          :name (text-id progress-fast-elevator)
+          :truthy-text (text-id progress-on)
+          :falsey-text (text-id progress-off)
+          :get-value-fn (lambda () (-> *pc-settings* fast-elevator?))
+          :on-confirm (lambda ((val symbol))
+            (set! (-> *pc-settings* fast-elevator?) val)
+            (commit-to-file *pc-settings*)))
+        (new 'static 'menu-generic-boolean-option
+          :name (text-id progress-fast-progress)
+          :truthy-text (text-id progress-on)
+          :falsey-text (text-id progress-off)
+          :get-value-fn (lambda () (-> *pc-settings* fast-progress?))
+          :on-confirm (lambda ((val symbol))
+            (set! (-> *pc-settings* fast-progress?) val)
+            (commit-to-file *pc-settings*))))))
```

Comment:
Can you actually move these to only show up for game options in the title screen? I don't want to deal with potential bugs of changing this mid-elevator and stuff (not that I believe there are any, but yeah).